gen/plugins/com.salesforce/com.salesforce.plugin.smartstore.js

```diff
@@ -389,7 +389,7 @@ var moveCursorToNextPage = function (isGlobalStore, cursor, successCB, errorCB)
     if (checkFirstArg(arguments)) return;
     var newPageIndex = cursor.currentPageIndex + 1;
     if (newPageIndex >= cursor.totalPages) {
-        errorCB(cursor, new Error("moveCursorToNextPage called while on last page"));
```

Comment:
To be adpatable into a promise, the error callback needs to return only one object (the reason of the failure).